MIT.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+
+<html lang="en-US">
+    <head>
+        <title>MIT License</title>
+
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link href="/@css/documentation_style.css" rel="stylesheet" type="text/css">
+    </head>
+
+    <body>
+        <h2>Copyright 2021, <a href="https://www.linkedin.com/in/ajffg1024">Ant&oacute;nio Jos&eacute; Gomes</a></h2>
+
+        <p>Permission is hereby granted, free of charge, to any person obtaining a copy of this <strong>PROJECT</strong> and associated documentation files, to deal in the <strong>PROJECT</strong> without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the <strong>PROJECT</strong>, and to permit persons to whom the <strong>PROJECT</strong> is furnished to do so, subject to the following conditions:</p>
+
+        <p>The above copyright notice and this permission notice shall be included in all copies or substantial portions of the <strong>PROJECT</strong>.</p>
+
+        <p>THE <strong>PROJECT</strong> IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE <strong>PROJECT</strong> OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].</p>
+    </body>
+</html>
+
+
```

Comment:
[OPTIONAL]

- [ ] MIT.html file can be removed as it has no value to your project. 
**Make sure to keep in your project MIT.md file.**

Reply:
Although this is an exercise, this is a web content.
I have setup an actual web space for deployment of this "application" (https://projects.microverse.datanet-pt.net/Hello-Microverse), and this file exists so the MIT license has a valid html form to be visible by the browser. It is linked from the html version of the README file -- at:  https://projects.microverse.datanet-pt.net/Hello-Microverse/README.html
I have setup this web space as I DO TEST it in an actual webserver, to which content is feeded by a git clone of the repository (as stated in the README.md file)